### DISCUSSION TITLE
Is there any way to use ssh on chromebook if it is blocked in terminal?

### DISCUSSION THREAD

**Discussion by s0urce-c0de** [3kh0/ext-remover#1135] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have an idea, but it requires ssh and ssh is blocked in terminal :(.

**Comment by cellebr8** [comment]
[This](https://chromewebstore.google.com/detail/secure-shell/iodihamcpbpeioajjeobimgagajmlibd) extension is by google and so was in (at least my) allowlist by default :)

**Reply by s0urce-c0de** [reply, marked as answer]
sorry, but it didn't work. thx for trying
